bento2seldon/bento.py

```diff
@@ -222,7 +222,14 @@ def _send_feedback(
         routing: Optional[int],
     ) -> None:
         if reward is not None:
-            self._monitor.observe_reward(reward)
+            extra = {}
+            if request is not None and hasattr(self, "extra_monitor_request_info"):
```

Comment:
Instead of checking for an attribute, it's better to make it more explicit, like creating a method with @property that returns `[]` by default. This way, it's a matter of overwriting a method to add properties. Also, I think another name would be better, like `extra_monitoring_request_fields`.